packages/arb-token-bridge-ui/src/components/TransactionHistory/FailedTransactionsWarning.tsx

```diff
@@ -1,61 +0,0 @@
-/*
```

Comment:
Q: Are we completely getting rid of this feature now?

Reply:
For now, we can add it right away after. We just need a good way of storing failed txs because right now we store pending and refetch status